packages/host/src/Host.tsx

```diff
@@ -197,7 +197,7 @@ const Host: FC = () => {
                           overflow="hidden"
                         >
                           <AppBar />
-                          <Box display="flex" flex={1} overflow="hidden">
+                          <Box display="flex" flex={1} overflow="scroll">
```

Comment:
didn't seem to damage the other pages - and now if the widgets flow to another row, the scrollbar shows